src/rpc/rawtransaction_util.cpp

```diff
@@ -21,10 +21,15 @@
 
 CMutableTransaction ConstructTransaction(const UniValue& inputs_in, const UniValue& outputs_in, const UniValue& locktime, bool rbf)
 {
-    if (inputs_in.isNull() || outputs_in.isNull())
-        throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid parameter, arguments 1 and 2 must be non-null");
+    if (outputs_in.isNull())
```

Comment:
`[rpc] walletcreatefundedpsbt: allow inputs to be null`:

Style-nit: add `{}` while at it, since you're touching this line.

Reply:
Will change in the followup in a separate commit, in case people think it's too late to touch.